sdks/java/core/src/test/java/org/apache/beam/sdk/io/CompressedSourceTest.java

```diff
@@ -235,6 +315,30 @@ public void testReadConcatenatedGzip() throws IOException {
     assertEquals(Bytes.asList(expected), actual);
   }
 
+  /**
+   * Using Lzo Codec Test a concatenation of lzo files is correctly decompressed.
+   *
+   * <p>A concatenation of lzo files as one file is a valid lzo file and should decompress to be the
+   * concatenation of those individual files.
```

Comment:
I think we need an enclosing `</p>`, or we can simply remove the `<p>` tag. Either way, can you please ensure `testReadConcatenatedGzip()` and `testReadMultiStreamBzip2()` follow the same javadocs format?

Reply:
This is happening when we run the spotlessApply task. When the </p> tag is clossed, the spotlessCheck fails. Not sure of the reason behind that.

Reply:
The closing </p> tag isn't needed in javadoc even if your editor is inserting it.